config/pwa.ts

```diff
@@ -36,10 +38,20 @@ export const pwa: ModuleOptions = {
   },
   workbox: {
     globPatterns: ['**/*.{js,css,html,txt,png,ico,svg}'],
-    navigateFallbackDenylist: [/^\/api\//],
+    navigateFallbackDenylist: [/^\/api\//, /^\/hi\//],
     navigateFallback: '/',
     cleanupOutdatedCaches: true,
     runtimeCaching: [
+      {
+        handler: 'NetworkFirst',
+        urlPattern: ({ sameOrigin, url }) => sameOrigin && url.pathname.startsWith('/hi/'),
+        options: {
+          plugins: [{
+            handlerDidError: async () => Response.redirect('/hi', 302),
+            cacheWillUpdate: async () => null,
+          }],
+        },
+      },
```

Comment:
It it possible to identify dynamic routes automatically in the Nuxt module? This is a starter template after all, hard code those path isn't really helpful

Reply:
https://github.com/vite-pwa/nuxt/pull/66